src/Symfony/Component/Lock/Test/StoreTestCase.php

```diff
@@ -19,7 +19,7 @@
 /**
  * @author Jérémy Derussé <jeremy@derusse.com>
  */
-abstract class AbstractStoreTestCase extends TestCase
+abstract class StoreTestCase extends TestCase
```

Comment:
This change should be reverted as we always prefix abstract classes with `Abstract`.